test/org/sagebionetworks/bridge/models/schedules/ScheduleTest.java

```diff
@@ -36,26 +36,28 @@ public void canRountripSerialize() throws Exception {
         schedule.setExpires(Period.parse("P2D"));
         schedule.setStartsOn(DateTime.parse("2015-02-02T10:10:10.000Z"));
         schedule.setEndsOn(DateTime.parse("2015-01-01T10:10:10.000Z"));
-        schedule.setEventId("eventId");
+        schedule.setEventId(Schedule.EVENT_ID_PROPERTY);
         schedule.setInterval(Period.parse("P3D"));
-        schedule.setLabel("label");
+        schedule.setSequencePeriod(Period.parse("P3W"));
```

Comment:
Should also add a test that we can parse sequencePeriod back from JSON.

Reply:
Added to ScheduleTest.